.github/workflows/ci.yml

```diff
@@ -39,19 +39,19 @@ jobs:
 
       - name: Install Redis Enterprise operator
         run: |
-          curl --silent --fail https://raw.githubusercontent.com/RedisLabs/redis-enterprise-k8s-docs/v6.0.12-5/bundle.yaml | kubectl apply -f -
+          curl --silent --fail https://raw.githubusercontent.com/RedisLabs/redis-enterprise-k8s-docs/v6.0.20-12/bundle.yaml | kubectl apply -f -
           kubectl rollout status deployment/redis-enterprise-operator --watch --timeout=5m || kubectl describe pod -lname=redis-enterprise-operator
 
       - name: Install cluster
-        timeout-minutes: 5
+        timeout-minutes: 10
```

Comment:
From past pipeline executions the average time seems to be about 7 minutes.  10 minutes should give enough time overall.